docs/operations/cli.rst

```diff
@@ -25,6 +25,10 @@ following are the command line options that Envoy supports.
 
   *(optional)* The output file path where the admin address and port will be written.
 
+.. option:: --local-address-ip-version <string>
+
+  *(optional)* The local address IP version. The options are ``v4`` or ``v6``. Defaults to ``v4``.
```

Comment:
It's used in a variety of places, but the most obvious one is it controls what is appended to XFF. So maybe mention that one?